amsd/__init__.py

```diff
@@ -1,6 +1,14 @@
+from __future__ import unicode_literals, absolute_import, division, print_function
```

Comment:
We don't need `from __future__ import` anymore. None of our apps runs on py2 anymore, so the new baseline is py3.5.